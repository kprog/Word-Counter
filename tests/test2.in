Olipa kerran maailma, jossa asui kaloja. Kerran maailmaan ilmestyi sukellusvene.
Osaavatko sukellusveneet uida? Josseivat osaa,
niin ne uppoavat pohjaan.

