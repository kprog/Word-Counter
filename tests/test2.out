asui 1
ilmestyi 1
jossa 1
josseivat 1
kaloja 1
kerran 1
kerran 1
maailma 1
maailmaan 1
ne 1
niin 1
olipa 1
osaa 1
osaavatko 1
pohjaan 1
sukellusvene 1
sukellusveneet 1
uida 1
uppoavat 1

sentences: 4
3 word sentences: 1
4 word sentences: 1
5 word sentences: 0
6 word sentences: 2
